lib/utils.pm

```diff
@@ -2826,4 +2827,21 @@ sub define_secret_variable {
     script_run("set +a");
 }
 
+=head2 ping_size_check
+    ping_size_check($target);
+ping_size_check will ping defined target with different and increasing size with disabled
+packet fragmentation.
+=cut
+
+sub ping_size_check {
+    my $target = shift;
```

Comment:
I suggest to enable signatures and:
```suggestion
sub ping_size_check ($target) {
```
(If it is not too much trouble to enable signatures for this module.)

Reply:
I prefer old style :)

Reply:
… until new Perl versions will eventually enforce it :)